From 48c3af6fdc4c73186d91aa126ce48ee28bdbb99f Mon Sep 17 00:00:00 2001
From: "Gavin T. Olson" <gavster@gmail.com>
Date: Sat, 3 Feb 2018 13:31:32 -0500
Subject: [PATCH 3/3] Added indi-ssag to top-level CMakeLists

---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6cde3700..60158c1c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -79,6 +79,7 @@ option(WITH_RTLSDR "Install RTL-SDR Receiver" On)
 option(WITH_RADIOSIM "Install RadioSim Receiver" On)
 option(WITH_GPSNMEA "Install GPS NMEA Driver" On)
 option(WITH_ARMADILLO "Install Armadillo & Platypus Driver" On)
+option(WITH_SSAG "Install Orion Starshoot Autoguider Driver" On)
 
 # Add/remove cases for OSX
 IF (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
@@ -280,6 +281,11 @@ if (WITH_RADIOSIM)
 add_subdirectory(indi-radiosim)
 endif (WITH_RADIOSIM)
 
+# Starshoot autoguider
+if (WITH_SSAG)
+add_subdirectory(indi-ssag)
+endif (WITH_SSAG)
+
 #endif (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 
 
-- 
2.16.0

